example/zeto/README.md

```diff
@@ -0,0 +1,36 @@
+# Example: ZKP Fungible Tokens with Zeto
+
+This example demonstrates a privacy-preserving fungible tokens using [Zeto](https://github.com/hyperledger-labs/zeto) on Paladin.
+
+See the [tutorial](https://lf-decentralized-trust-labs.github.io/paladin/head/tutorials/zeto/) for a detailed explanation.
```

Comment:
This tutorial doesn't exist yet, does it?

Reply:
added